packages/protocol/jest.config.js

```diff
@@ -0,0 +1,12 @@
+//
+// Copyright 2020 DXOS.org
+//
+
```

Comment:
it's the same config as in "jest" section in pacakge.json. Just copy all of the options from here to there and delete this file